user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_components.md

```diff
@@ -53,6 +53,11 @@ The I-DOCSIS EPM Solution includes the following dashboards:
 - I-DOCSIS Channel Utilization
 - I-DOCSIS Network
 - I-DOCSIS Service Group [Fiber Node]
+- I-DOCSIS Service Group [Fiber Node] - CM DS SNR OOS
+- I-DOCSIS Customer Diagnosis tool-CM Info
```

Comment:
Just to be sure: Is it correct that there are no spaces around the "-" in the dashboard name in line 57 and 58? It looks a bit odd and is inconsistent with the spelling in line 56.

Reply:
Hi, no is not correct, it was my mistake. Should I correct and do the pull again?

Reply:
I've adjusted this. OK to merge as it is now?